notes/transcript15.md

```diff
@@ -213,108 +213,228 @@ Igor: I guess the signature won't signatures won't match but we can do it a shot
-Oskar: So I would suggest we move on. I think this requires maybe more effort. Maybe we could put together some different proposals and then have some pros and cons and also then we can bring it up in next core dev call. Because I think that there are several ideas and that go either ways but they might need to be fleshed out a bit more whether it's sort of the timeframe recoverability or if we do a breaking change then should we have a path to recoverability, in some way we can limit the scope by creative solutions and so on. I think there is different sort of there's multiple options here that we can explore but it probably requires asynchronous effort. 
-Ricardo: Yeah I agree with you. I just wanted to know who we should talk with to discuss this later on. 
-Oskar: So I imagine Igor might want to do a few more runs on their post-mortem or whatever you want to call it pre-mortem, I don't know what you gonna call it, but then we can bring it up next core dev call and then we might we can also bring it up in a protocol call if that's interesting but I think.
-Ricardo: So basically use the Disqus post in this meeting 
+
+Oskar: So I would suggest we move on. I think this requires maybe more effort. Maybe we could put together some different proposals and then have some pros and cons and also then we can bring it up in next core dev call. Because I think that there are several ideas and that go either ways but they might need to be fleshed out a bit more.
+
+### 1:05:00 - 1:10:00
+
+Oskar: whether it's sort of the timeframe recoverability or if we do a breaking change then should we have a path to recoverability, in some way we can limit the scope by creative solutions and so on. I think there is different sort of there's multiple options here that we can explore but it probably requires asynchronous effort.
+
+Ricardo: Yeah I agree with you. I just wanted to know who we should talk with to discuss this later on.
+
+Oskar: So I imagine Igor might want to do a few more runs on the post-mortem or whatever you want to call it pre-mortem, I don't know what you gonna call it, but then we can bring it up next core dev call and then we might we can also bring it up in a protocol call if that's interesting but I think.
+
+Ricardo: So basically use the Disqus post in this meeting.
+
 Igor: yeah I think Disqus is a good start and then we'll figure out from there. Because it's already a discussion started so there is no point to switch it somewhere else.
```

Comment:
```suggestion
Igor: yeah I think Discuss is a good start and then we'll figure out from there. Because it's already a discussion started so there is no point to switch it somewhere else.
```